pkg/kubernetes/deploy.js

```diff
@@ -105,39 +103,55 @@ define([
                         namespaces.push(jdata);
                     } else {
                         text = _("Unsupported Entity. ");
-                        file_note.show().text(text);
+                        manifest_file_details.show().text(text);
+                        manifest_file_note.addClass('has-error').show();
                         return;
                     }
                 }
             } else {
                 text = _("Unable to read the Kubernetes application manifest file. ");
-                file_note.show().text(text);
+                manifest_file_details.show().text(text);
+                manifest_file_note.addClass('has-error').show();
                 return;
             }
 
-            create_everything(namespace, services, rcs, pods)
-                .progress(function(code, response, n, total) {
-                    //TODO Progress bar
-
-                })
-                .done(function() {
-                    /* code gets run when everything is created */
-                    disable_deploy_button();
-                    hide_progress_message();
-                     $('#deploy-app-dialog').modal('hide');
+            show_progress_message();
+            deploy_btn.prop("disabled", true);
 
+            client.get_services()
```

Comment:
I need to check if kubernetes daemons are up and the apiserver is functioning well by calling a REST api.
and if if it works create all entities.